app/uk/gov/hmrc/ngrraldfrontend/models/components/NGRRadio.scala

```diff
@@ -28,19 +28,14 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.Radios
 trait RadioEntry
 
 sealed trait YesNoItem extends RadioEntry
-case object Yes extends YesNoItem
-case object No extends YesNoItem
+case object `true` extends YesNoItem
+case object `false` extends YesNoItem
 
```

Comment:
Imho, it looks odd. We should find a way to avoid using restricted words this way